db/database.go

```diff
@@ -943,11 +943,11 @@ func (db *Database) UpdateAllDocChannels(doCurrentDocs bool, doImportDocs bool)
 		}
 		var err error
 		if db.UseXattrs() {
-			err = db.Bucket.WriteUpdateWithXattr(key, KSyncXattrName, 0, func(currentValue []byte, currentXattr []byte) (raw []byte, rawXattr []byte, err error) {
+			err = db.Bucket.WriteUpdateWithXattr(key, KSyncXattrName, 0, func(currentValue []byte, currentXattr []byte, cas uint64) (raw []byte, rawXattr []byte, err error) {
```

Comment:
I'd prefer to leave this as-is.  I agree it's getting a bit wide, but in a standard editor it's still readable, and overall I find the inline function tracks better visually than a separate callback, particularly with the non-xattr usage right below it.